ops/health-check-route.s.md

```diff
@@ -0,0 +1,73 @@
+# [Standard] Writing a health check route
+
+## Owner: Arthur Levoyer
+
+# Why
+
+In order to monitor correctly their environments, backend services require a HealthCheck route which returns a status depending on how the API is running.
+
+## Checks
+
+- [ ] Make a call to every database used by the API
+- [ ] Send a 2xx status code status in case of API running correctly and 5xx status code if not
```

Comment:
Send a 2xx status code status if the API is running correctly, 5xx status code if not